bouter/bout_stats.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+from numba import jit
+
+
+@jit(nopython=True)
+def bout_stats(vigor, tail_sum, bouts, wnd_turn_pts):
```

Comment:
I would name this function with an underscore, it will be called by a method that works with dataframes, right?